doc/articles/features/compass.md

```diff
@@ -32,6 +32,16 @@ If you are planning to use the `HeadingTrueNorth`, your app must declare `androi
 [assembly: UsesPermission("android.permission.ACCESS_FINE_LOCATION")]
 ```
 
+> [!Note]
+> Android lacks a dedicated API for accessing the compass heading. Uno Platform utilizes the accelerometer and magnetometer sensors to calculate the magnetic north heading, a method endorsed by Google.
+
+> In rare cases, you may encounter inconsistent results due to the need for sensor calibration. The process of recalibrating the compass on Android varies across phone models and Android versions. To recalibrate, consult online resources specific to your device. Here are two links that may assist you in recalibrating the compass:
+
+> [Google Help Center: Find and improve your location’s accuracy](https://support.google.com/maps/answer/2839911)
+> [Stack Exchange Android Enthusiasts: How can I calibrate the compass on my phone?](https://android.stackexchange.com/questions/10145/how-can-i-calibrate-the-compass-on-my-phone)
+
+> It's also important to note that concurrently running multiple sensors in your app may impact sensor speed.
```

Comment:
@morning4coffe-dev The note syntax look odd in the doc: 
<img width="1010" alt="image" src="https://github.com/unoplatform/uno/assets/5839577/71a91bd7-f36b-4a55-bfd1-3e7f0e82953b">

Did you intend to put it as a single "note" outline?

Reply:
Oh, thanks for noticing @jeromelaban! Yes, I have installed a VS Code extension which should help me with this a little. The > should be continuous, right?

Reply:
Should be good now 
![image](https://github.com/unoplatform/uno/assets/78549750/e9e4e4c2-1b0c-4e5e-998a-d037f0a18570)